homeassistant/components/fyta/config_flow.py

```diff
@@ -63,3 +65,54 @@ async def async_step_user(
         return self.async_show_form(
             step_id="user", data_schema=DATA_SCHEMA, errors=errors
         )
+
+    async def async_step_reauth(
+        self, entry_data: Mapping[str, Any]
+    ) -> ConfigFlowResult:
+        """Handle flow upon an API authentication error."""
+        self._entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])  # pylint: disable=attribute-defined-outside-init
```

Comment:
Please define this as class attribute instead of disabling the pylint rule
Just below `VERSION` add: `_entry: ConfigEntry | None = None`

Reply:
Thank you for the helpful instruction!